config/default/kustomization.yaml

```diff
@@ -123,14 +113,6 @@ replacements:
       kind: Certificate
       name: serving-cert
       namespace: kserve
-  - fieldPaths:
-    - metadata.annotations.[cert-manager.io/inject-ca-from]
```

Comment:
Should this be removed?

Reply:
You are right. I will rollback this part and file.